recipe-server/client/actions/preference-experiment/index.js

```diff
@@ -43,6 +43,7 @@ export default class PreferenceExperimentAction extends Action {
             preferenceName,
             preferenceValue: branch.value,
             preferenceBranchType,
+            preferenceType,
```

Comment:
CircleCI is failing partially for reasons unknown to me, but also because we need to update the tests for the action to assert that this parameter is passed as well.